kpi/tasks.py

```diff
@@ -95,15 +95,13 @@ def enketo_flush_cached_preview(server_url, form_id):
     )
     response.raise_for_status()
 
-
-@celery_app.task
-def remove_asset_snapshots(asset_id: int):
+@celery_app.task(time_limit=82800, soft_time_limit=82800)
```

Comment:
Maybe comment with a human readable time. Do we really need both? We won't need to catch SoftTimeLimitExceeded. If you do end up keeping both I would set a constant with a variable name that denotes the human readable time.

Reply:
@jamesrkiger, you may want to look at `settings.CELERY_LONG_RUNNING_TASK_TIME_LIMIT` and `settings.CELERY_LONG_RUNNING_TASK_SOFT_TIME_LIMIT`